src/main/java/redis/clients/jedis/json/parser/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * This package contains the classes and interfaces related to JSON processing module.
+ */
+package redis.clients.jedis.json.parser;
```

Comment:
After update, now it seems we don't need a separate package. `util` package could be an option to put the interface and default class; but I'm leaning more towards `json` package.

Reply:
I also think that the `json` package it's a better place.